defaults/main.yml

```diff
@@ -2,6 +2,7 @@
 composer_path: /usr/local/bin/composer
 composer_keep_updated: false
 composer_version: ''
+composer_version_branch: '--1'
```

Comment:
I would prefer to see '--2' as default value even is this means you have to adopt your config for older projects.

Reply:
Hi,

Done.
Beware though because Composer 2 is still very new and using it right now might be breaking plugins.